persistent/Database/Persist/Quasi.hs

```diff
@@ -919,71 +919,19 @@ module Database.Persist.Quasi
     , PersistSettings
     , upperCaseSettings
     , lowerCaseSettings
-    -- ** Getters and Setters
-    , module Database.Persist.Quasi
```

Comment:
hmm does this reduce the interface of the module? may be good to look at a diff of the generated haddocks. `stack haddock --open` for here + the haddocks on persistent hackage

Reply:
Looks like the new interface is a superset of the old one, as it should be.